mypy_django_plugin/transformers/models.py

```diff
@@ -361,11 +365,11 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
                 fallback_manager_info = self.get_or_create_manager_with_any_fallback()
                 if fallback_manager_info is not None:
                     assert self.model_classdef.info.self_type is not None
-                    self.add_new_node_to_model_class(
-                        manager_name,
-                        Instance(fallback_manager_info, [self.model_classdef.info.self_type]),
-                        is_classvar=True,
+                    manager_type = Instance(
+                        fallback_manager_info,
```

Comment:
Let's please move this to a helper function / method.

Reply:
True, good point